website/src/types/config.ts

```diff
@@ -62,6 +62,11 @@ export const inputField = z.object({
     name: z.string(),
     displayName: z.string().optional(),
     noEdit: z.boolean().optional(),
+    required: z.boolean().optional(),
+    definition: z.string().optional(),
+    example: z.union([z.string(), z.number()]).optional(),
+    guidance: z.string().optional(),
```

Comment:
I know so far none of the other properties are documented, but I'd really like some documentation on "guidance" and also "definition" (at first I thought it's some technical definition).

but not a blocking thing.